README.md

```diff
@@ -115,3 +115,11 @@ OpenZeppelin exists thanks to its contributors. There are many ways you can part
 ## License
 
 OpenZeppelin is released under the [MIT License](LICENSE).
+
+[API docs]: https://docs.openzeppelin.org/v2.3.0/api/token/erc721
+[guides]: https://docs.openzeppelin.org/v2.3.0/get-started
```

Comment:
Note that these links will not show up on the readme, they are simply there to making linking to these URLs easier (e.g. you only need to do [forum] as opposed to \[forum](link))